code_generation/code.cpp

```diff
@@ -170,7 +170,7 @@ void Code::addFormattedText(const QString &text)
     int lineLength = 0;
 
     QString line;
-    const QStringList words = text.split(QLatin1Char(' '), QString::SkipEmptyParts);
+    const QStringList words = text.split(QLatin1Char(' '), Qt::SkipEmptyParts);
```

Comment:
I re-did this in commit e660167 with a Qt version ifdef so as not to raise the min Qt version for now.